packages/phoenix-evals/src/phoenix/evals/models/mistralai.py

```diff
@@ -128,8 +118,8 @@ def _rate_limited_completion(self, **kwargs: Any) -> Any:
         @self._rate_limiter.limit
         def _completion(**kwargs: Any) -> Any:
             try:
-                response = self._client.chat(**kwargs)
-            except self._MistralAPIException as exc:
+                response = self._client.chat.complete(**kwargs)
+            except Exception as exc:
```

Comment:
we shouldn't be catching generic exceptions here, is the Mistral SDK no longer returning typed exceptions?

Reply:
looks like the new exception is `models.SDKError`

you can replace `_MistralAPIException` with:

```
from mistralai.models import models.SDKError

# if an SDKError is raised, check that it's a rate limit error:
except SDKError as exc:
    if exc.status_code == 429:
        raise MistralRateLimitError() from exc
```

This needs to be applied in both the sync and async methods

Reply:
Fixed!